terraform/10-video-hearing-wowza/modules/wowza/vm_base.tf

```diff
@@ -47,7 +48,7 @@ resource "azurerm_linux_virtual_machine" "wowza" {
     publisher = local.publisher
     offer     = local.offer
     sku       = local.sku
-    version   = "latest"
+    version   = local.version
```

Comment:
why can't we use latest?

Reply:
yeah we can, was changed when we thought 'latest' may have been causing the issue with the VM not booting.

Reply:
can we change back to latest.
Wowza seem to update then remove the previous so this will just make sure we are alway up to date.